packages/snaps-cli/src/utils/snap-config.ts

```diff
@@ -1,10 +1,39 @@
-import { readFileSync } from 'fs';
+import path from 'path';
+import type browserify from 'browserify';
 import { Arguments } from 'yargs';
 import yargsParse from 'yargs-parser';
 import yargs from 'yargs/yargs';
 import builders from '../builders';
-import { logError } from './misc';
-import { CONFIG_FILE } from '.';
+import { CONFIG_FILE, logError } from './misc';
+import { isSnapConfig } from './snap-config.__GENERATED__';
+
+/** @see {isSnapConfig} ts-auto-guard:type-guard */
+export type SnapConfig = {
```

Comment:
Hmm, do we want to nest all the current options in `options`? Is there a reason they can't be defined next to `bundler`?

Reply:
I.e. 
```
const brfs = require('brfs');
module.exports = {
  port: 8084,
  transpilationMode: "localOnly",
  bundler: (bundler) => bundler.transform(brfs)
}
```

Reply:
I specifically don't want to pollute the config with both command line options config and built-in config in the same namespace. We would need to watch how we name our cli options in the future